src/binary/paths.ts

```diff
@@ -11,7 +11,10 @@ const BUNDLE_SUFFIX = getBundleSuffix();
 export async function setBinaryRootPath(
   updatedPath: vscode.Uri
 ): Promise<void> {
-  binaryRootPath = path.join(updatedPath.fsPath, "binaries");
+  binaryRootPath =
+    process.env.NODE_ENV === "test"
```

Comment:
lets extract it to some helper, it's used in several places

Reply:
or even better, there is a `extensionMode` property on the `ExtensionContext` that we can use to determine if it is a test run. `NODE_ENV === test` can be modified locally, ref https://github.com/codota/tabnine-vscode/issues/325